### DISCUSSION TITLE
Slack Integration

### DISCUSSION THREAD

**Discussion by DaveSanders** [dailydotdev/daily#968] [Feature Requests]
I'd like to have either an API (preferred) or a Slack bot that can simply create a slack message whenever a certain query of posts comes up with a new item. (i.e. if I want to follow the AI squad in a team's slack)

**Comment by idoshamun** [comment]
Hi @DaveSanders,

Thanks for the suggestion! API is something that we constantly consider, but it's not prioritized at the moment.
Stay tuned for more

**Comment by capJavert** [comment]
**Update: daily.dev now has a public API** 🎉

This one's been asked for here for a long time, so: it's live, and it's
available to everyone. Generate a personal access token at
[daily.dev/settings/api](https://daily.dev/settings/api) and start calling it.

What's in v1:

- **Feeds** — your personalized For You feed, trending posts, filtering by tag or source
- **Posts** — full post details, AI summaries and comment threads
- **Search** — posts, tags and sources
- **Bookmarks** — read and manage your saved content
- **Notifications** — your activity feed and unread counts
- **Profile and reading streak** — your experience, tech stack and streak

📣 The announcement: https://daily.dev/posts/public-api-access-is-now-open-for-everyone-ljj8kh9sa
📚 Docs: https://docs.daily.dev/public-api/
